package.json

```diff
@@ -74,19 +74,20 @@
     "lodash.camelcase": "4.3.0",
     "minimist": "^1.2.5",
     "ngx-bootstrap-icons": "^1.7.2",
-    "ngx-markdown": "12.1.0",
-    "ngx-toastr": "^13.2.0",
+    "ngx-highlightjs": "^4.1.1",
```

Comment:
ngx-highlightjs was removed previously because 4.1.1 depended on a vulnerable package and was otherwise not doing anything. Is there a reason it is being re-introduced?

Reply:
It was added by conflict removed it.